CONTRIBUTING.md

```diff
@@ -56,6 +56,25 @@ Button.propTypes = {
 };
 ```
 
+### TypeScript
+
+### Type definitions for Props and States
+Always use `interface` for type definitions Props and States. Interfaces are more “extendable” due to the support of declaration merging. It is useful when components supports native HTML props, for example:
```

Comment:
Little grammar check

Always use `interface` when defining types for props and states. Interfaces are more "extendable" because of support for declaration merging. This is useful when components support native HTML props, for example: